helm/aragorn/values.yaml

```diff
@@ -116,11 +116,15 @@ redis:
   image:
     repository: redis
     tag: "7.0.12"
+  replicaCount: 2
```

Comment:
is there a reason why we have 2 replicas?

Reply:
Oh I have removed it, let me update it